examples/v1/workflows/cancellations.go

```diff
@@ -0,0 +1,45 @@
+package v1_workflows
+
+import (
+	"errors"
+	"time"
+
+	"github.com/hatchet-dev/hatchet/pkg/client/create"
+	v1 "github.com/hatchet-dev/hatchet/pkg/v1"
+	"github.com/hatchet-dev/hatchet/pkg/v1/factory"
+	"github.com/hatchet-dev/hatchet/pkg/v1/workflow"
+	"github.com/hatchet-dev/hatchet/pkg/worker"
+)
+
+type CancellationInput struct{}
+type CancellationResult struct {
+	Completed bool
+}
+
+func Cancellation(hatchet v1.HatchetClient) workflow.WorkflowDeclaration[CancellationInput, CancellationResult] {
+
+	// Create a task that sleeps for 10 seconds and checks if it was cancelled
+	cancellation := factory.NewTask(
+		create.StandaloneTask{
+			Name: "cancellation-task",
+		}, func(ctx worker.HatchetContext, input CancellationInput) (*CancellationResult, error) {
```

Comment:
nit on SDK consistency: is having `ctx` be first a Go convention? if it is, let's leave this. but if it's not, the context is the second arg in python + ts

Reply:
yes, it is apparently